guides/v2.1/coding-standards/docblock-standard-general.md

```diff
@@ -425,7 +430,7 @@ In general, use the `@throws` tag when the code uses *throw*:
  *
  * @param  string $elementId
  * @param  string $attribute
- * @param  mixed  $value
+ * @param  int|string|float|bool|object|null  $value
```

Comment:
Why is this change needed? It is not exactly equivalent transformation since `mixed` is broader.

Reply:
This was a request by @orlangur - I personally think `mixed` is fine, but don't feel strongly either way.

Reply:
@paliarush do you mean it can be `resource`? Generally `mixed` does not smell well.

Reply:
Besides `resoruce`, the `@mixed` annotation includes any future types, too.